libgnucash/core-utils/core-utils.i

```diff
@@ -56,6 +56,15 @@ void
 %newobject gnc_path_get_bindir;
 gchar * gnc_path_get_bindir(void);
 
+%newobject gnc_path_get_localedir;
+gchar * gnc_path_get_localedir(void);
+
+/* Name of our gettext-domain (defined in config.h) */
+%constant char* GETTEXT_PACKAGE = GETTEXT_PACKAGE;
+
+/* Gettext availability */
+%constant int HAVE_GETTEXT = HAVE_GETTEXT;
+
```

Comment:
I see no benefit here. It is set together with its relatives in https://github.com/Gnucash/gnucash/blob/c38c4557a56370a51236903e326f4429fd5b2caf/CMakeLists.txt#L700 ff
But if no (recent) version is found line 259 or 263 will abort CMake.

@gjanssens am I missing something her? Or should we remove all occurences of HAVE_GETTEXT like constants. It seems to me like a workaround, when certain feature were not available in some old versions of gettext or intltool.

Reply:
Another aspect: That check was intended for the C. I have several questions about Python:

If we talk about Python, https://en.wikipedia.org/wiki/CPython is meant on all platforms?

As it is part of their standard libs we need no tests in CMake for existence and/or compatibility?

Reply:
It's intended to support [Iron Python](https://ironpython.net/) too, but that also has gettext as a standard module. CMake tests won't do any good because python's gettext is needed at runtime, not build-time.

Reply:
@fellen I have to redefine the constant to be able to access it from Python. I could also include config.h and access it directly but it seemed easier this way. If the flag is not necessary for python then that's a different issue. In that case this could be omitted as well.

Reply:
@c-holtermann As far as I understand it now, it is part of cPython's standard libs as it is in C's glib since a very long time. That has not been the case, when GnuCash has been a guile program "decades ago".

i am considering to remove HAVE_BIND_TEXTDOMAIN_CODESET, HAVE_DCGETTEXT and HAVE_GETTEXT from CMakeLists.txt, config.h[.in] and gnucash-bin.c, which is the only user, as it is only confusing today.
@jralls , @gjanssens any objections?

Reply:
@fellen go ahead. Only HAVE_GETTEXT is even used and it's effectively always defined: ```if (UNIX or MINGW)``` is all we support at present. If you're so motivated just clean out all of those settings.

Reply:
This Gettext flags were removed in commit 7cb4c1d.

Reply:
@fellen no worries. I'm glad if work on one part leads to work on the bigger picture. Thanks. I'll adapt my files.